financial-templates-lib/logger/Transports.js

```diff
@@ -0,0 +1,53 @@
+// This module generates transport objects for the winston logger to push messages to.
+// Primarily this module separates the logic for reading in state environment variables
+// from the logger itself. All Winston transport objects and their associated formatting
+// are created within this module.
+
+// Transport objects
+const ConsoleTransport = require("./ConsoleTransport");
+const SlackTransport = require("./SlackTransport");
+const TwilioTransport = require("./TwilioTransport");
+
+require("dotenv").config();
+const argv = require("minimist")(process.argv.slice(), {});
+
+// transports array to store all winston transports
+let transports = [];
+
+// add a console transport to log to the console.
+transports.push(ConsoleTransport.createConsoleTransport());
+
+// If there is "test" in the environment then skip the slack or twilio transports.
+if (argv._.indexOf("test") == -1) {
```

Comment:
At some point, we'll have to give in fully to dependency injection. I.e., configure which transports are active for a Logger in `index.js`. That'll avoid the production code branching on `isTest`.

Reply:
Drive-by comment: totally agree -- @chrismaree is it reasonable to go ahead and go full dependency injection going? Or is that too big an overhaul for this PR?

Reply:
that should be a seperate PR I think @mrice32 . This is already a fair amount of changes so I think we can leave that until later. will submit a pr shortly after this one gets merged.